processing/src/main/java/org/apache/carbondata/processing/datatypes/ArrayDataType.java

```diff
@@ -131,6 +131,10 @@ public void getAllPrimitiveChildren(List<GenericDataType> primitiveChild) {
     }
   }
 
+  public GenericDataType getChildren() {
```

Comment:
It is better to add it in `GenericDataType` and override it here

Reply:
@jackylk getChildren() added to GenericDataType and overriden.